src/components/PageSizeSelector/index.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from "@emotion/styled";
+import { Select, Option } from "@leafygreen-ui/select";
+import { PAGE_SIZES } from "constants/index";
+import usePageSizeSelector from "./usePageSizeSelector";
+
+interface Props {
+  value: number;
+  "data-cy"?: string;
+  onChange?: (i: number) => void;
+}
+
+/**
+ * `data-cy` prop is not currently supported by @leafygreen-ui/select
+ */
```

Comment:
I'm confused by this comment because the `data-cy` prop is still being used. Do you mean that we're including it anyway so that the prop is in place once LeafyGreen supports it?

Reply:
Yeah that was the intention but i just removed it all together and replaced it with a spread.